src/Native/Runtime/PalRedhawk.h

```diff
@@ -849,6 +849,8 @@ REDHAWK_PALIMPORT Int32 __cdecl _stricmp(const char *string1, const char *string
 #if defined(_X86_) || defined(_AMD64_)
 REDHAWK_PALIMPORT uint32_t REDHAWK_PALAPI getcpuid(uint32_t arg1, unsigned char result[16]);
 REDHAWK_PALIMPORT uint32_t REDHAWK_PALAPI getextcpuid(uint32_t arg1, uint32_t arg2, unsigned char result[16]);
+REDHAWK_PALIMPORT uint32_t REDHAWK_PALAPI xmmYmmStateSupport();
```

Comment:
Should these two be just one PAL function?

Reply:
I lifted this from CoreCLR where it's two methods and each of them only has a single callsite like here. I can fix this up if you insist, but this code currently matches the relevant section of codeman.cpp in CoreCLR.

Reply:
ok, sounds good.